core/types/sbundle.go

```diff
@@ -0,0 +1,113 @@
+package types
+
+import (
+	"errors"
+	"math/big"
+	"sync/atomic"
+
+	"github.com/ethereum/go-ethereum/common"
+	"golang.org/x/crypto/sha3"
+)
+
+var (
+	ErrIncorrectRefundConfig = errors.New("incorrect refund config")
+)
+
+type SBundle struct {
```

Comment:
could we add a small concise godoc on `SBundle` ?

Specifically what the `S` denotes, and how it's different from a regular bundle or transaction